### DISCUSSION TITLE
How to utilize notificationAuthToken in callback API for notification?

### DISCUSSION THREAD

**Discussion by Nish0808** [camaraproject/QualityOnDemand#226] [Q&A] [answered]
In QOD API, while creating QoS Session Resource, the request body parameter contains one optional attribute that is "notificationAuthToken". I believe this should be used for callback API for notification but I was looking for some more clarity here, like , should we send it as a bearer token or basic http authentication token ? 

Thanks in advance!

**Comment by eric-murray** [comment, marked as answer]
Hi @Nish0808

Use of `notificationAuthToken` is described in the [CAMARA API Design Guidelines](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#instance-based-implicit-subscription).

These guidelines should be read in conjunction with all API definitions, for otherwise we would need to copy these into every API definition's documentation, which would be cumbersome. Note that these guidelines have recently been updated, so QoD is yet to fully catch up, but will do so in the near future.

**Reply by Nish0808** [reply]
Thank you @eric-murray . This helps a lot!
